### PR TITLE
Release script

### PR DESCRIPTION
**Pull request by SamBarker**
Add a script for running releases of the extension.

### CODE REVIEW

File: scripts/release.sh [hunk 1]
```diff
@@ -0,0 +1,95 @@
+#
```
[the remark this answers is not in the document]
Comment (k-wall) [reply scripts/release.sh:2]: Sam fixed this already.

[the remark this answers is not in the document]
Comment (SamBarker) [reply scripts/release.sh:2]: Oh I was hoping the bot would auto resolve it.

File: scripts/release.sh [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+if [[ -z ${RELEASE_VERSION} ]]; then
+  echo "No version specified aborting"
+  exit 1
+fi
+
+GREEN='\033[0;32m'
+NC='\033[0m' # No Color
+
+git stash --all
```
Comment (k-wall) [inline scripts/release.sh:51]: I've normally favoured re-cloning the repo to a temporary location and running the release from there in order to eliminate the risk of pollution from a developer's own workspace.  You are doing a git stash, but what if I have local commits? What do you think?

**Review by k-wall — APPROVED** [review]
I've not executed the script myself, but it all looks good to me.  thank you @SamBarker

File: scripts/release.sh (hunk 2 above)
Comment (SamBarker) [reply scripts/release.sh:51]: I didn't think too deeply but I thought `stash` plus branch from `remote/branch` would avoid the local commits issue. See [#L55](https://github.com/kroxylicious/kroxylicious-junit5-extension/pull/77/files#diff-1bba00e5aca52286a667c09eff92ba2ca8e3ca77e0d31b0599cc829ab0173389R55) 

Do you have a sample of how you've managed a clean checkout in the past?

Comment (k-wall) [reply scripts/release.sh:51]: I don't have any example directly to hand.  I didn't notice the remote/branch.  You approach is good enough.

**Pull request merged**